com.unity.netcode.gameobjects/Components/HalfVector3.cs

```diff
@@ -13,45 +14,70 @@ namespace Unity.Netcode.Components
     /// </remarks>
     public struct HalfVector3 : INetworkSerializable
     {
+        public const int Size = 3;
```

Comment:
Is this `Size` or `Length`?

`Size` kind of implies size in the memory or on the network, usually in bytes but it's not it right?
`Length` is implying "how many items does this container contain?" and that's 3 items — and 3 items x 2 bytes = 6 bytes in total.

Maybe a nit-pick but wanted to bring it up (same applies to `HalfVector4` too)

Reply:
or in general, maybe never expose this constant value to public since we're using it internally for the `for` loops and 3 for HalfVector3 and 4 for HalfVector4 are kind of obvious by their name — what do you think?